Lab1

A1zen4 opened the pull request.

A1zen4 committed:
Add files via upload

vozniuk-lntu commented:
1. нема обмеження на мерж без апрува
2. нема деплою
3. чому один коміт? Це типу відразу маркер що скопіював все та одним комітом збережено

A1zen4 committed:
Update index.html

Edited index.html file. Removed <div> tags (rechecked requirements to 1st Lab, decided to remove them because there are not so many elements to separate them by using the <div>tag)

A1zen4 committed:
Update index.html

Edited second "Ольга Рєпіна", should be "Frontend Developer"
Also added a deploy and site link to the repository description

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,81 @@
+<footer>
+    <a href="">WebStudio</a>
+        <ul>
+            <li><a href="https://goo.gl/maps/qWS8BV7t2WGqXTAa9">м. Київ, пр-т Лесі Українки, 26</a></li>
```
Посилання на сторонній ресурс, загуглити атрибут `rel`

vozniuk-lntu replied:
не правильно
нофолов норефер

vozniuk-lntu commented on index.html:
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html lang="uk">
+<html> 
+<head>
```
head розгорни ВСКодом бо тут не вистачає важливих мета тегів

vozniuk-lntu reviewed (changes requested):
1. коли посилання пусте заглушкою є href="#"
2. в мейн чотири секції має бути, об'єднай теги в секції

A1zen4 committed:
Update index.html

Розгорнув head meta тегами;
поставив "#" в пусті "a href";
поставив атрибут rel="external", оскільки посилання на сторонній ресурс;
додав 4 секції

vozniuk-lntu reviewed (approved):
окей
виправ там міні правку

A1zen4 committed:
Update index.html

changed rel = external to rel = nofollow noreferrer
